### DISCUSSION TITLE
multivariate forecasting using sktime

### DISCUSSION THREAD

**Discussion by nyk2001** [alan-turing-institute/sktime#3774] [General]
Hi All, 

I am new to sktime. I am using the following Kaggle dataset to forecast sales for different stores and items:

![image](https://user-images.githubusercontent.com/3079053/201475160-3e737662-bd38-4930-9d77-2fc888b5080d.png)

I have created x_train, y_train, x_test, and y_test datasets. You can see the first 5 rows for x_train and y_train datasets: 

![image](https://user-images.githubusercontent.com/3079053/201475183-86614215-7967-4a2a-a6bd-31adaca10df5.png)

Train and test datasets are similar in terms of schema. I want to **train a model using **date, store, and item features**** and make predictions. 

**I am looking for an example**, where I can use SkTime to train **Arima, Prophet, and LightGBM** models and make predictions using x_test dataset. I will then compare the performance of these models using MAPE metric.

Any help or guidance will be really appreciated.

Thank

  [1]: https://i.stack.imgur.com/1wLL3.png
  [2]: https://i.stack.imgur.com/S3kkf.png
  [3]: https://i.stack.imgur.com/GvwgU.png

**Comment by nyk2001** [comment]
> Hi All,
> 
> I am new to sktime. I am using the following Kaggle dataset to forecast sales for different stores and items:
> 
> ![image](https://user-images.githubusercontent.com/3079053/201475160-3e737662-bd38-4930-9d77-2fc888b5080d.png)
> 
> I have created x_train, y_train, x_test, and y_test datasets. You can see the first 5 rows for x_train and y_train datasets:
> 
> ![image](https://user-images.githubusercontent.com/3079053/201475183-86614215-7967-4a2a-a6bd-31adaca10df5.png)
> 
> Train and test datasets are similar in terms of schema. I want to **train a model using **date, store, and item features**** and make predictions.
> 
> **I am looking for an example**, where I can use SkTime to train **Arima, Prophet, and LightGBM** models and make predictions using x_test dataset. I will then compare the performance of these models using MAPE metric.
> 
> Any help or guidance will be really appreciated.
> 
> Thank

I have found the example. Thanks

**Reply by fkiraly** [reply]
for completeness: there should be examples in `01_forecasting.ipynb`, see here in binder, https://mybinder.org/v2/gh/sktime/sktime/main?filepath=examples
